rero_ils/modules/items/api.py

```diff
@@ -899,3 +899,14 @@ def organisation_view(self):
         """Get Organisation view for item."""
         organisation = Organisation.get_record_by_pid(self.organisation_pid)
         return organisation['view_code']
+
+    def item_has_active_loan_or_request(self):
+        """Return True if active loan or a request found for item."""
+        states = ['PENDING'] + \
```

Comment:
Sometimes better to write with `format`

Reply:
In this part of code, we concatenate two arrays.
Using "format" is not possible.
Maybe I'm wrong.